src/FloatingContainer/FloatingContainer.tsx

```diff
@@ -86,9 +86,9 @@ class FloatingContainer extends React.PureComponent<FloatingContainerProps, Floa
   }, this.props.windowResizeDebounceDelay)
 
   onClickOut = (e: MouseEvent) => {
-    e.stopPropagation()
+    const isOutside = this.floatingContainerRef.current && !this.floatingContainerRef.current.contains(e.target)
```

Comment:
Nice! I was not aware of [Node.contains](https://developer.mozilla.org/en-US/docs/Web/API/Node/contains) TIL